clients/python/Makefile

```diff
@@ -116,8 +116,8 @@ _check_venv_active:
 
 .PHONY: install-dev
 install-dev: _check_venv_active .install-dev-reqs python-client  ## installs osparc_client and osparc in edit mode
-	uv pip install -e artifacts/client
-	uv pip install -e .
+	# for some reason this command refuses to run with uv. Looks related to https://github.com/astral-sh/uv/issues/1661
+	pip install -e artifacts/client -e .
```

Comment:
Show me.
I think this can be done just by adding `@touch setup.py` before ... It happened to me

Reply:
or check the syntax in osparc-simcore for installing editable packages. it's all used there.

Reply:
Yes, but it seems this is not because the package is editable, but because it depends on another package which is also editable and I am specifying this on the same line (which I should do to ensure they are compatible).

Reply:
I added `--editable`